rfcs/text/0005-host-metric-fields.md

```diff
@@ -1,8 +1,8 @@
 # 0005: Host Metric Fields
 <!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
 
-- Stage: **1 (proposal)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
-- Date: **2020-08-21** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+- Stage: **2 (draft)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **2020-10-14** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
```

Comment:
I think I forgot to adjust this date when I merged stage 1 yesterday. Will open a quick PR to fix this.

But you don't need to adjust this here. It's meant to capture the date an RFC was merged at a given stage :-)

For now you could even put "TBD" again here.